src/sso/pages/AppDetails.tsx

```diff
@@ -88,7 +88,7 @@ const AppDetails = () => {
           </div>
 
           <div style={styles.logout}>
-            <TButton startIcon={<LogoutIcon></LogoutIcon>}>Logout</TButton>
+            <TButton startIcon={<LogoutIcon></LogoutIcon>} onClick={async () => {await api.ExternalUser.logout()}}>Logout</TButton>
```

Comment:
this will not work. `logout()` is a member function of an externalUser object, not a class function. You will need to call it on the user object.

Perhaps we need to add the user object to a state storage in zustand which is shared between components.